dojo/sla_config/views.py

```diff
@@ -0,0 +1,99 @@
+# #  product
+import logging
+
+from django.contrib import messages
+from django.contrib.auth.decorators import login_required
+from django.http import HttpResponseRedirect
+from django.shortcuts import render
+from django.urls import reverse
+
+from dojo.authorization.authorization import user_has_configuration_permission_or_403
+from dojo.authorization.authorization_decorators import user_is_configuration_authorized
+from dojo.forms import SLAConfigForm
+from dojo.models import SLA_Configuration, System_Settings
+from dojo.utils import add_breadcrumb
+
+logger = logging.getLogger(__name__)
+
+
+@user_is_configuration_authorized('dojo.add_sla_configuration', 'superuser')
```

Comment:
This seems a little too restrictive imo. I think allowing owner roles to touch these settings would be a bit better

Reply:
Owner roles exist only for authorization of objects like Products or Findings. This is the correct way for authorization of a configuration. The `superuser` bit is only legacy and will have to be removed after #6446 is merged.